src/commands/removeMark.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { createRichTextEditor } from "../core/create"
+import { describe, expect, it, jest } from "@jest/globals"
+import { findByText, getByText, waitFor } from "@testing-library/dom"
+import { removeMark } from "./removeMark"
+import { simulateSelectionByText } from "./fixtures/helpers"
+import { testEditorStateWithMarks, testSchema } from "./fixtures/test-state"
+
+describe("removeMark", () => {
+  const onChange = jest.fn()
+  const attributes = { "aria-labelledby": "label-text-123" }
+
+  it("removes the given <strong> Mark from a paragraph in the document", async () => {
+    const node = document.createElement("div")
+    const { dispatchTransaction } = createRichTextEditor({
+      node,
+      onChange,
+      attributes,
+      initialEditorState: testEditorStateWithMarks,
+    })
+    const currentStrongTags = node.querySelectorAll("strong").length
+    // Check that there is at least one strong tag
+    expect(currentStrongTags).toBeGreaterThan(0)
```

Comment:
I've had to update this so that the tests won't be as brittle if the data from fixtures changes. Basically add 2 Strong  of the same type (in this case <strong>) was breaking this test, so I use the total number of marks as the base line.